stuff/pipelines/test_contest_callback.py

```diff
@@ -12,19 +12,20 @@
 import time
 
 from typing import List
+import omegaup.api
 
 import contest_callback
 import test_credentials
 import rabbitmq_connection
 import test_constants
-import omegaup.api
 
 
 sys.path.insert(
     0,
     os.path.join(
         os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
 import lib.db   # pylint: disable=wrong-import-position
+from stuff.pipelines.contest_callback import Ranking
```

Comment:
Intenta realizar el import de la siguiente forma para ver si deja de aparecer uno de los errores:
```suggestion
from contest_callback import Ranking
```